doc/contextUML.txt

```diff
@@ -0,0 +1,31 @@
+# Copy and paste the following into http://www.plantuml.com/plantuml/uml/
+
+@startuml
+class Config
+class Context
+class Model
+class StateMap
+class State
+class FaultState
+class IdleState
+class InMotionState
+class OfflineState
+class PauseState
+class StandbyState
+class StartupState
+
+Config ..> Context
+Config ..> Model
```

Comment:
Why `Config` depends on the `Model` instead of the reversed way?

Reply:
Fixed